config/reader.go

```diff
@@ -610,6 +610,10 @@ func processConfig(unprocessedConfig *Config, fromCloud bool, logger logging.Log
 		}
 	}
 
+	if err := logging.UpdateLoggerRegistry(cfg.LogConfig); err != nil {
+		return nil, err
+	}
```

Comment:
let's add a small test for this in `config/reader_test.go` - e.g. reading a config with a `LogConfig` and ensuring that `GetRegisteredLoggerNames` returns the corresponding names if successful.

Reply:
done!